cfgov/v1/models/sublanding_filterable_page.py

```diff
@@ -48,10 +49,13 @@ class SublandingFilterablePage(FilterableFeedPageMixin, FilterableListMixin, CFG
 
     template = 'sublanding-page/index.html'
 
+    objects = PageManager()
```

Comment:
Yeah, this is somewhat non-intuitive. [This](http://docs.wagtail.io/en/v1.6.3/releases/1.6.1.html#multi-level-inheritance-and-custom-managers) is the part of Wagtail release notes for 1.6.1 that motivated this. There's a somewhat long discussion on the changes in Django and Wagtail that motivated this change [here](https://github.com/torchbox/wagtail/issues/2933).